tests/resources/test-types.json

```diff
@@ -868,9 +868,7 @@
       "large",
       "small"
     ],
-    "forVehicleConfiguration": [
-      "rigid"
-    ],
+    "forVehicleConfiguration": null,
```

Comment:
Where does this change come from?

Reply:
The Voluntary Test is the root for both Vitesse 100 and Vitesse 100 replacement which require both rigid and articulated supported. Changing to null allows the support for both vehicle configurations. All other test types already have rigid defined explicitly.